Keen.NET.Test/Keen.NET.Test.csproj

```diff
@@ -83,6 +83,12 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="..\SharedAssemblyInfo.cs">
+      <Link>Properties\SharedAssemblyInfo.cs</Link>
```

Comment:
They are there: https://github.com/keenlabs/keen-sdk-net/pull/43/files#diff-de185a19d58a705bb8adced8c3dab3d8

They are in the root of the solution and linked in for each project, so you won't actually see them in the ...\Properties\ folders on disk.